docs/tutorials/flight_controller_telemetry_tutorial.rst

```diff
@@ -0,0 +1,395 @@
+which will keep the module spinning for an extended period, making it easier to check the telemetry.
+
+
+.. figure:: ../_static/tutorial_images/fc_telemetry_tutorial/mp_motor_test.png
+    :align: center
+    :width: 50%
+    :alt: Motor Test in Mission Planner
+
+    Motor Test in Mission Planner to Test Telemetry When Spinning
+
+The image below shows telemetry data for a spinning module. Note the non-zero RPM value. For more information on how the flight controller calculates
+the displayed RPM value based on the value reported by the module, see the :ref:`arducopter_erpm_vs_rpm` section below.
+
+.. figure:: ../_static/tutorial_images/fc_telemetry_tutorial/mp_spin_telemetry.png
+    :align: center
+    :width: 50%
+    :alt: Live Telemetry Updates in Mission Planner When Spinning
+
+    Live Telemetry Updates in Mission Planner When Spinning
+
+.. _arducopter_erpm_vs_rpm:
+
+ERPM vs. RPM
+##############
+As of v4.3.6, ArduCopter flight controllers expect the DSHOT telemetry messages to match the `KISS ESC standard <https://www.rcgroups.com/forums/showatt.php?attachmentid=8524039&d=1450424877>`_.
+That means that it expects the module to send ERPM/100, while Vertiq modules directly send RPM as covered in :ref:`ESC Telemetry <manual_telemetry>`.
+
+To convert from ERPM to RPM, the ERPM value should be divided by the pole count of the motor divided by 2. The flight controller is expecting to receive 
+ERPM/100, so when it receives telemetry it tries to calculate RPM using the following formula shown below. This calculation can also be
+seen in the `ArduCopter code for BLHeli telemetry parsing <https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_BLHeli/AP_BLHeli.cpp#L1447>`_.
+
+.. figure:: ../_static/tutorial_images/fc_telemetry_tutorial/px4_erpm_to_rpm_formula.png
+    :align: center
+    :width: 40%
+    :alt: ERPM to RPM Conversion
+
+    ERPM to RPM Conversion
+
+Because Vertiq modules send RPM instead of ERPM, this conversion is unnecessary, and actually leads to the flight controller calculating the incorrect RPM by default.
+
+As discussed in the :ref:`fc_telemetry_erpm_to_rpm` section for PX4 below, it is possible to workaround this conversion issue by setting the number of motor poles to 200.
+However, as of v4.3.6, ArduCopter flight controllers limit the maximum value of SERVO_BLH_POLES to 127. So it is not possible to apply this workaround when using
+ArduCopter. The RPM value calculated by the flight controller will not be the true RPM value of the module because of this, but any logged telemetry data can be converted
+based on the formula above to get the actual RPM value.
+
+Standard PWM Telemetry
+***********************
+Vertiq modules support sending ESC telemetry when using :ref:`Standard PWM <hobby_standard_pwm>` or :ref:`other analog protocols <hobby_other_protocols>`

```

Comment:
and not or between :ref:`Standard PWM <hobby_standard_pwm>` or :ref:`other analog protocols <hobby_other_protocols>`